frontend/create-group/src/components/steps/General.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <article>
+        <h1>Jouw vereniging aansluiten</h1>
```

Comment:
Should we already start by implementing language support? But only support Dutch for now. Not applicable in this PR, but in a different one.

Reply:
I would skip langauge support for now, it's not that much work to add it later.